lib/core/connection_factory/http.rb

```diff
@@ -5,7 +5,9 @@
 module Core
   module ConnectionFactory
     class Http
-      def self.build(url, timeout: 5, open_timeout: 5, retries: 2)
+      TIMEOUT = ENV.fetch('FRONTEND_HTTP_REQUEST_TIMEOUT').to_i
```

Comment:
Yes, I am happy. I don't think anyone will put banana or similar in the http request timeout env var. And yes, I prefer to be strictly to have the env set before the app started than having bugs exploding during app execution.